### DISCUSSION TITLE
How to start the timer in reverse direction ?

### DISCUSSION THREAD

**Discussion by vishnuborn1996** [ndresx/react-countdown#117] [General]
How to start the timer in reverse direction ?

**Comment by ndresx** [comment]
Hi, somebody has posted a possible solution here https://github.com/ndresx/react-countdown/issues/40 by adjusting the `now` prop. Otherwise, you could try to play around with the `overtime` prop if that's what you are looking for.

**Reply by vishnuborn1996** [reply]
Thanks much

**Comment by carlos-dubon** [comment]
```jsx
const someDate = new Date('2020-12-18T00:00:00Z');

<Countdown
    date={new Date(0)}
    now={() => - new Date().valueOf() + someDate.valueOf()}
    overtime
/>
```
